src/Widgets/calendar/Calendar.vala

```diff
@@ -19,20 +19,24 @@
 
 namespace DateTime.Widgets {
     public class Calendar : Gtk.Box {
-        ControlHeader heading;
-        CalendarView cal;
+        public ControlHeader heading;
```

Comment:
Are you sure these need to be public and not private?

Reply:
Only `ControlHeader heading;` should be, fixing it.